clients/metadata/metadata_cassandra.go

```diff
@@ -189,26 +189,40 @@ type CassandraMetadataService struct {
 // interface implementation check
 var _ m.TChanMetadataService = (*CassandraMetadataService)(nil)
 
+func parseConsistency(cfgCons string) (lowCons gocql.Consistency, midCons gocql.Consistency, highCons gocql.Consistency) {
+
+	// use a minimum default consistency of 'One'
+	lowCons, midCons, highCons = gocql.One, gocql.One, gocql.One
+
+	switch cons := strings.Split(cfgCons, ","); len(cons) {
+	case 1:
+		highCons = gocql.ParseConsistency(strings.TrimSpace(cons[0]))
+
+	case 2:
+		lowCons = gocql.ParseConsistency(strings.TrimSpace(cons[0]))
+		midCons = lowCons
```

Comment:
It is confusing: when two values [0] is low and [1] is high, when three values, values [0] is low and [1] is mid. Is this to keep backward compatibility?